view/src/components/surplus.js

```diff
@@ -119,10 +119,31 @@ const styles = (theme) => ({
     width: "100%",
   },
   table: {
-    marginTop: "50px",
+    marginTop: "48px",
+  },
+  formText: {
+    marginBottom: "16px",
   },
 });
 
+// Used as a placeholder since the CustomTable isn't connected to CRUD
+const tableData = {
+  columns: [
+    { title: "Role", field: "contactRole" },
+    { title: "Name", field: "contactName" },
+    { title: "Email", field: "contactEmail" },
+    { title: "Phone Number", field: "contactPhone" },
+  ],
+  data: [
+    {
+      contactRole: "Farm Manager",
+      contactName: "Jane Doe",
+      contactEmail: "jane@doe.com",
```

Comment:
Maybe we can do a more gender neutral name? Perhaps "Jamie" Doe?

Reply:
great suggestion!